pkg/agent/ippool/ippool.go

```diff
@@ -1,17 +1,20 @@
 package ippool
 
 import (
-	"fmt"
-
 	"github.com/sirupsen/logrus"
 
 	networkv1 "github.com/harvester/vm-dhcp-controller/pkg/apis/network.harvesterhci.io/v1alpha1"
 	"github.com/harvester/vm-dhcp-controller/pkg/util"
 )
 
 func (c *Controller) Update(ipPool *networkv1.IPPool) error {
+	if !networkv1.CacheReady.IsTrue(ipPool) {
+		logrus.Warning("ippool is not ready")
```

Comment:
please add ippool namespace and name into log, others are similar

Reply:
Updated. PTAL, thanks